app/routes/packet_processor.js

```diff
@@ -109,13 +114,12 @@ class PacketProcessor {
     participants.forEach(async (uId) => {
       const userNodeData = await this.sessionRepository.getUserNodeData(uId);
       if (!userNodeData?.length) {
-        this.isAllowedForOfflineStorage(
-          packetsMapOrPacket[uId] || packetsMapOrPacket
-        ) &&
-          this.perationsLogRepository.savePacket(
-            uId,
-            packetsMapOrPacket[uId] || packetsMapOrPacket
-          );
+        const uPacket = packetsMapOrPacket[uId] || packetsMapOrPacket;
+        this.isAllowedForOfflineStorage(uPacket) &&
+          this.operationsLogRepository.savePacket(uId, uPacket);
+
+        this.pushNotificationsRepository.sendPushNotification(uId, uPacket);
```

Comment:
Let's say we send a message to group chat with 5 people, and all 4 (except you) are offline.
In this case we will initiate 4 push tasks in queue.

It make send to inititate a single push task to queue, but to pass an array of recipients